semcore/color-picker/src/index.d.ts

```diff
@@ -35,7 +35,7 @@ export type ColorsProps = BoxProps & {
   /**
    * Array of color items
    */
-  colors?: string[];
+  colors?: Array<string | null>;
```

Comment:
Can this change affect users somehow? How does it related to storybook?

Reply:
this is real type, how we (or others) use it. 
for now, we check our code for examples with ts and that's why I added this.